.github/workflows/linux.yml

```diff
@@ -231,7 +231,7 @@ jobs:
         cmake --build build --parallel 2
         ctest --test-dir build --output-on-failure
 
-  gcc9_py38_pd_nompi_h5_ad2_libcpp:
+  gcc9_py38_pd_nompi_h5_ad2_libcpp_julia:
```

Comment:
Let's document the new optional dependencies here in docs:
- https://openpmd-api.readthedocs.io/en/0.15.1/dev/dependencies.html
- https://github.com/openPMD/openPMD-api/#dependencies